docs/backends/last9.mdx

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
please also add it to `mint.json` [here](https://github.com/odigos-io/odigos/blob/281af99307a62e86b6b1eccdca6ebb796d9541f2/docs/mint.json#L202) so it will show up in side menu